docs/docs/manual/exporting.md

```diff
@@ -6,34 +6,97 @@ sidebar_label: Exporting
 
 ## Overview
 
+Once your data is cleaned, you will need to get it out of OpenRefine and into the system of your choice. OpenRefine outputs a number of file formats, can upload your data directly into Google Sheets, and can export directly to Wikidata. 
+
+You can also export your full project data so that it can be opened by someone else using OpenRefine (or yourself, on another computer).
 
 ## Export data
 
 Note you will only export data in the current view - that is, with current filters and facets applied. 
 
+To export from a project, click the Export button at the top right corner and pick the format you want. You options are:
 
-
-*   TSV/CSV
-*   HTML table
+*   Tab-separated value (TSV) or Comma-separated value (CSV)
+*   HTML-formatted table
 *   Excel (XLS or XLSX)
 *   ODF spreadsheet
-*   Google Sheets \
+*   Upload to Google Sheets (requires Google account authorization)
+*   [Custom tabular exporter](#custom-tabular-exporter)
+*   [SQL statement exporter](#sql-statement-exporter)
+*   [Templating exporter](#templating-exporter)
 
-*   Custom tabular export
-*   SQL
-*   Templating export \
+You can also [export reconciled data to Wikidata](wikidata#editing-wikidata-with-openrefine), or save your schema for future use with other OpenRefine projects:
 
 *   Upload edits to Wikidata
 *   Export to QuickStatement
```

Comment:
I think this should be plural `Quickstatements`. Do we also need to mention which version of Quickstatements is supported?

Reply:
It produces QuickStatements commands in version 1.